astronomer/providers/amazon/aws/example_dags/example_emr_sensor.py

```diff
@@ -72,7 +71,7 @@
     cluster_creator = EmrCreateJobFlowOperator(
         task_id="create_job_flow",
         job_flow_overrides=JOB_FLOW_OVERRIDES,
-        emr_conn_id=EMR_CONN_ID,
+        aws_conn_id=AWS_CONN_ID,
```

Comment:
will this fail for apache-airflow-providers-amazon < 4.1.0 ? We should make it fail safe for previous versions of the provider then. Something similar to this, with a try-except block catching the exception: https://github.com/astronomer/astronomer-providers/pull/471#discussion_r907184079

Reply:
hmm, it would fail for apache-airflow-providers-amazon < 4.1.0. Does it make sense to safe fail here since this is an integration test?

Reply:
@phanikumv @bharanidharan14 what is your opinion on this?

Reply:
I am not sure if we can use try-catch to an argument being passed to an operator.

Reply:
> I am not sure if we can use try-catch for an argument being passed to the operator.

Reply:
Perhaps we can add a comment in the example DAG that based on the versions of the provider what arguments are expected

Reply:
@pankajastro adding comments in the example DAG is reasonable to me, so it helps user to know what arguements needs to be passed

Reply:
@pankajastro Is this review comment is addressed ?